external/fv3fit/tests/test_pack.py

```diff
@@ -57,6 +57,19 @@ def test_roundtrip(
         np.testing.assert_array_equal(unpacked[name].values, ds[name].values)
 
 
+def test_pack_2d_and_3d():
```

Comment:
I thought this might be a bug reproducer but then it passed, I figure might as well keep it.